app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :username, presence: true, length: { in: 4..12 }, uniqueness: true
+
+  has_many :events
```

Comment:
- [ ] This should have a  `foreign_key: :creator_id` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

Reply:
The foreign_key, as the name indicates, comes from an external/foreign element. 
In this case, we utilize the user_id as a foreign_key due to the fact that the user_id exists as part of the User. 
"create" is an alias or a way to RE-NAME that relationship existing between the user and the event. 
So, if we write Event.first.create we will obtain the user that created this event. 
If we add a column named creator_id in the events table, it will not be able to accomplish the essence of a foreign key, 
since it will belong to the same event table, which doesn't fit the logic nor the purpose behind using a foreign_key. 
Also, the Odin Project specifications do not explicitly entail the name that must be used as the foreign_key, and in any case, then creator_id should belong to the user and not the event. Please see reference:
https://www.w3schools.com/sql/sql_foreignkey.asp

Reply:
- [ ] Please check the screenshot I provided .

Reply:
Please read this example, of RUBY GUIDES
** 3.5 Bi-directional Associations Ruby on Rails guides**
https://guides.rubyonrails.org/association_basics.html#bi-directional-associations.